src/sbvr-api/sbvr-utils.ts

```diff
@@ -1270,7 +1270,11 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 				>;
 				// Add/check the relevant permissions
 				try {
-					if (abstractSqlModel.tables[resolveSynonym($request)] == null) {
+					const resolvedResourceName = resolveSynonym($request);
+					if (
+						!resolvedResourceName.includes('#canAccess') &&
+						abstractSqlModel.tables[resolvedResourceName] == null
```

Comment:
I'd put this check first because I'd expect it to pass the majority of the time and therefore avoid running the second check most often